Assets/BossRoom/Prefabs/Dungeon/Dungeon Pieces/env_floor3.prefab

```diff
@@ -92,7 +92,7 @@ BoxCollider:
   m_GameObject: {fileID: 4371491365999748502}
   m_Material: {fileID: 0}
   m_IsTrigger: 0
-  m_Enabled: 1
```

Comment:
So from what I remember, the BoxCollider component on these floor tile prefabs should be enabled (and in-scene as well) to bake the NavMesh. A script on awake turns these components off, and the big level plane is used instead for the raycasts from click/touch.